### PR TITLE
Code Quality: Improved quick access widget

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   - This PR improves performance and memory leaks when the quick access widget is displayed multiple times.

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

File: src/Files.App/App.xaml.cs [hunk 1]
```diff
@@ -321,7 +321,7 @@ private async void Window_Closed(object sender, WindowEventArgs args)
 				SaveSessionTabs();
 				MainPageViewModel.AppInstances.ForEach(tabItem => tabItem.Unload());
 				MainPageViewModel.AppInstances.Clear();
-				await Task.Delay(100);
+				await Task.Delay(500);
```
Comment (hishitetsu) [inline src/Files.App/App.xaml.cs:324]: This is not directly related to the quick access widget, but I increased the wait time to ensure that open tabs are closed before sleep.

**Review by yaira2 — APPROVED** [review]
LGTM
